semcore/flags/src/countries.js

```diff
@@ -505,3 +505,21 @@ export const iso3iso2 = {
   ZMB: 'ZM',
   ZWE: 'ZW',
 };
+
+export const nameWithoutIso = {
+  ABK: 'Abkhazia',
```

Comment:
keys are very similar to iso2 but they are not(
Maybe we'll use more complete names